happytransformer/gen/trainer.py

```diff
@@ -72,6 +72,7 @@ def train(self, input_filepath, dataclass_args: GENTrainArgs):
             self._generate_json(dataclass_args.save_preprocessed_data_path, tokenized_dataset["train"], "train")
 
         self.logger.info("Training...")
+        print("tokenized_dataset['train']", tokenized_dataset['train'])
```

Comment:
Printing straight to `stdout` as a side effect in a library feels wrong

Reply:
Forgot to delete that: 
a211655